### PR TITLE
fixup recapitate passing of recombination_map

### PR DESCRIPTION
**Pull request by petrelharp**

### CODE REVIEW

File: pyslim/slim_tree_sequence.py [hunk 1]
```diff
@@ -461,6 +462,7 @@ def recapitate(self,
         else:
             ts = self
 
+        ts.dump("temp.trees")
```
Comment (jeromekelleher) [inline pyslim/slim_tree_sequence.py:465]: Did you mean to do this?

Comment (petrelharp) [reply pyslim/slim_tree_sequence.py:465]: no! thanks for the catch. You'll notice a force-push to `main` shortly after this PR was merged, so it's all good now: https://github.com/tskit-dev/pyslim/blob/0e4f0a187c7bc75c68a78593f541675d28e9c1a3/pyslim/slim_tree_sequence.py#L465

**Comment by grahamgower** [comment]
This looks good, thanks @petrelharp!

The stdpopsim 0.1.2 release works with the latest changes you've merged here, even with msprime 1.0.0a4 installed. So I think this is fine for you to roll a new pyslim release. If there are issues with the dev branch of stdpopsim, we'll make changes there in line with what you've suggested over in https://github.com/popsim-consortium/stdpopsim/issues/761#issuecomment-783496303

**Comment by petrelharp** [comment]
Ok - I am just waiting on one small thing on the SLiM side.

**Pull request merged**